com.cloudchewie.cloudotp.yaml

```diff
@@ -0,0 +1,158 @@
+app-id: com.cloudchewie.cloudotp
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
```

Comment:
why do you need vala?

Reply:
module ayatana-ido and libayatana-indicator need this

Reply:
Are you sure?

Reply:
they are written in C

Reply:
100%

```
-- Checking for module 'gobject-introspection-1.0'
--   Found gobject-introspection-1.0, version 1.82.0
CMake Error at /usr/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find Vala (missing: VALA_COMPILER)
Call Stack (most recent call first):
  /usr/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindVala.cmake:52 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/CMakeLists.txt:164 (find_package)


-- Configuring incomplete, errors occurred!
Error: module ayatana-ido: Child process exited with code 1
```